backend/application/syson-sysml-import/src/main/java/org/eclipse/syson/sysml/parser/ProxyResolver.java

```diff
@@ -38,81 +36,64 @@ public class ProxyResolver {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ProxyResolver.class);
 
-    public void resolveAllProxy(final EObject content) {
```

Comment:
When you change the APIs please add those breaking changes in the changelog/release notes.